src/toPlaylists.js

```diff
@@ -2,27 +2,24 @@ import { shallowMerge } from './utils/object';
 import { segmentsFromTemplate } from './segment/segmentTemplate';
 import { segmentsFromList } from './segment/segmentList';
 import { segmentsFromBase } from './segment/segmentBase';
-
 export const generateSegments = (segmentInfo, attributes) => {
-  if (segmentInfo.template) {
-    return segmentsFromTemplate(
-      shallowMerge(segmentInfo.template, attributes),
-      segmentInfo.timeline
+  if (segmentInfo.list) {
```

Comment:
Was there a specific reasoning for switching the order of `list` and `template` here? I think either order should be fine, just curious.

Reply:
I was just checking for the SegmentList output.